test/test_utility/test_utils.py

```
@@ -1,12 +1,113 @@
 import numpy as np
+import pandas as pd
 from numpy.testing import assert_allclose, assert_array_equal
-from MetricsReloaded.utility.utils import intersection_boxes, guess_input_style, com_from_box, point_in_box, point_in_mask, area_box, compute_box, compute_center_of_mass, compute_skeleton, combine_df, distance_transform_edt, one_hot_encode, median_heuristic, box_ior, box_iou, union_boxes, max_x_at_y_less, min_x_at_y_less, skeletonize, trapezoidal_integration
+from MetricsReloaded.utility.utils import MorphologyOps as MO
+from MetricsReloaded.utility.utils import intersection_boxes, guess_input_style, merge_list_df, com_from_box, point_in_box,point_in_mask, area_box, min_x_at_y_more, compute_box, compute_center_of_mass, combine_df, box_ior, box_iou, union_boxes, max_x_at_y_less, min_x_at_y_less
 
 
 box3 = [2,3, 4,4]
 box4 = [4,4,5,6]
 
+map = np.zeros([10,10])
```

Ruff (0.15.10)

[error] Variable `map` is shadowing a Python builtin

(A001)